modules/engines/engine-hdbti/src/main/java/com/sap/xsk/hdbti/dao/XSKCSVRecordDao.java

```diff
@@ -257,7 +257,7 @@ protected void setValue(PreparedStatement preparedStatement, int i, int dataType
             throws SQLException {
         logger.trace("setValue -> i: " + i + ", dataType: " + dataType + ", value: " + value);
 
-        if (value == null) {
+        if (value == null || value.isEmpty()) {
```

Comment:
Empty value is not always `null`, for text data types it should be allowed to insert `""` as an empty value not as `null`. For data types that require parsing (e.g. `DATE`, `TIME`, `TIMESTAMP`, etc.) `null` value should be set.